src/main/java/mate/academy/hibernate/relations/exception/DataProcessingException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.hibernate.relations.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message) {
+        super(message);
+    }
+
+    public DataProcessingException(String massage, Throwable e) {
```

Comment:
There is a typo in the parameter name 'massage'. It should be 'message' to be consistent with the standard naming convention.